pandas/core/resample.py

```diff
@@ -62,20 +62,6 @@ class Resampler(_GroupBy):
     _attributes = ['freq', 'axis', 'closed', 'label', 'convention',
                    'loffset', 'base', 'kind']
 
-    # API compat of allowed attributes
-    _deprecated_valids = _attributes + ['__doc__', '_cache', '_attributes',
-                                        'binner', 'grouper', 'groupby',
```

Comment:
whoops, didn't push the button probably :-)

My question was about the above `_deprecated_valids`: what was the purpose of them? At least before, those attributes did not raise any warning.

Reply:
But you can probably ignore my comment, those attributes are "hided" from tab completion anyway, and didn't change for the rest. I suppose this was here before to prevent raising any warning on them?

Reply:
yes, this was gymnastics to make all of this work